src/config.ts

```diff
@@ -42,23 +42,30 @@ export namespace config {
 }
 
 function init (credential: firebase.credential.Credential) {
-  let loaded: any;
+  let loadedFromFile = {};
+  let firebaseEnv = {};
+  if (process.env.FIREBASE_PROJECT) {
+    firebaseEnv = { firebase: JSON.parse(process.env.FIREBASE_PROJECT) };
+  }
+
   try {
-    loaded = require('../../../.runtimeconfig.json');
+    let path = process.env.CLOUD_RUNTIME_CONFIG || '../../../.runtimeconfig.json';
+    loadedFromFile = require(path);
   } catch (e) {
     try {
-      loaded = require('../../../config.json');
+      loadedFromFile = require('../../../config.json');
```

Comment:
This caused confusion with @samtstern's functions and was meant to be alpha behavior. Should we cut this or give a deprecation notice?